app/src/main/java/org/simple/clinic/patientcontact/CallPatientView.kt

```diff
@@ -26,10 +33,26 @@ class CallPatientView(
       secureCallingGroup.visibility = if (field) View.VISIBLE else View.GONE
     }
 
+  var agreedToVisitClicked: AgreedToVisitClicked? = { Toast.makeText(context, "WIP", Toast.LENGTH_SHORT).show() }
+
+  var remindToCallLaterClicked: RemindToCallLaterClicked? = { Toast.makeText(context, "WIP", Toast.LENGTH_SHORT).show() }
+
+  var removeFromOverdueListClicked: RemoveFromOverdueListClicked? = { Toast.makeText(context, "WIP", Toast.LENGTH_SHORT).show() }
+
+  var normalCallButtonClicked: NormalCallButtonClicked? = null
+
+  var secureCallButtonClicked: SecureCallButtonClicked? = null
```

Comment:
Please make these `private`.

Reply:
They need to be public so that they can be accessed by the `ContactPatientBottomSheet` to register listeners.

Reply:
Can we make that change when we get there since it isn't obvious as to why are they kept open?

Reply:
I'd rather not do this now because all the work is already done and making this change now will cause conflicts when I rebase my branch.

If you feel strongly about this, I can make this change though.